charter/index.md

```diff
@@ -0,0 +1,228 @@
+---
+title: Font and Text Community Group Charter
+---
+This charter:  {{ page.url | remove: "index.html" | prepend: site.url }}
+
+Feedback about this document:
+    https://github.com/w3c/font-text-cg/issues
+
+## Goals
+
+The Font and Text community group gathers individuals and organizations
+interested in discussing and developing specifications and implementations
+for technologies
+which operate on and at the interface between
+text encoding and font formats.
+Examples of such technologies are
+shaping engines and text layout applications.
+
+## Scope of work
+
+Under the present charter, the Community Group will
+determine a structure and process
+under which such specifications may be created
+by which the greatest number of interested parties may be able to participate.
+Until such a structure and process is determined,
+no technical development of specifications or implementations
+will be undertaken by this Community Group
+and no technical contributions will be accepted.
+
+## Deliverables
+
+No technical specifications will be produced by this Community Group
+and no technical contributions are
+to be solicited
+or accepted.
+
+* This Community Group aims to produce a proposed Charter,
+to be used to set up a technical group,
+possibly a W3C Working Group,
+under which members of this community should conduct technical work.
+
+* If, after producing such a charter,
+the Community Group determines that its current composition is adequate
+for conducting its work,
+then the charter produced is
+to be adopted as an updated Charter for this CG.
+
+* If the Community Group determines that it would be appropriate
+to divide the technical work into several venues,
+it may produce several such proposed charters
+rather than a single one.
+
+* Various supporting documents and articles
+describing the problem space
+may also be produced.
+
+All deliverables of this Community Group will use the
+[W3C Software and Document License](http://www.w3.org/Consortium/Legal/2015/copyright-software-and-document).
+
+This group maintains an [index](https://w3c.github.io/font-text-cg/work-items/)
+of the documents it is working on.
+
+## Community and Business Group Process and Patent Policy
+
+The group operates under the [Community and Business Group Process](https://www.w3.org/community/about/agreements/).
+Terms in this charter that conflict with those of the Community and Business Group Process are void.
+
+As with other Community Groups,
+W3C seeks organizational licensing commitments
+under the [W3C Community Contributor License Agreement (CLA)](http://www.w3.org/community/about/agreements/cla/).
+(Technical Proposals in this Community Group charter are applicable “Specification” in the CLA).
```

Comment:
I don't think we can drop this, this helps define how to handle things that may turn up in issues or other communications that wouldn't otherwise be clear. This is just another way of saying things that are said elsewhere ("technical specifications") but I think it's worth the verbiage to suggest that all technical proposals even if they aren't technically for specification documents are handled this way.